docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition.md

```diff
@@ -624,6 +624,162 @@ Here is the guide to transition from legacy model preprocessing to new API prepr
 
             - Not available in OVC tool. Please check Python API.
 
+Cutting Off Parts of a Model
+############################
+
+In new conversion API input and output model cutting is no longer available. It is recommended to perform cutting in the original framework instead.
+Below are examples of input and output cutting for TensorFlow protobuf, TensorFlow SavedModel, and ONNX formats in legacy conversion API, including examples of how to perform identical cut in the original framework.
+For PyTorch, TensorFlow 2 Keras, and PaddlePaddle, it is recommended to change the original model code to perform the model cut. 
+
+``TensorFlow protobuf format / tf.Graph / tf.GraphDef``
+#######################################################
+
+.. list-table::
+   :header-rows: 1
```

Comment:
I think it would be better to have a top-bottom layout of the comparison, as opposed to left-right columns. The code doesn't really align anyways.

Reply:
Done.